torch2trt/converters/cast.py

```diff
@@ -0,0 +1,72 @@
+from torch2trt.torch2trt import *
+from torch2trt.module_test import add_module_test
+
+
+def convert_cast(ctx):
+    """
+    A simple converter for supporting casting operations.
+
+    IMPORTANT: Note that because TensorRT does not support
+    64 bit data types, .long() will not be supported
+    """
+    input_tensor = ctx.method_args[0]
+    layer = ctx.network.add_identity(input_tensor._trt)
+    output = ctx.method_return
```

Comment:
In all the cases, IMO, all the layers are getting converted to an identity layer at FP32. 

This might not work for lower precisions such as INT8 and FP16

We need something like this 

`layer.precision = trt.int8` for int8 and something similar for fo16

Reply:
Thank you for the useful feedback.  

You are right. I did not factor in lower precisions (INT8 and FP16). 
I will update the code, run the tests and report back after committing.

Reply:
@SrivastavaKshitij I have made the suggested updates. If you have any additional suggestions / recommendations, I will take those into consideration and make the appropriate changes. 

Thank you for taking time to make a review.